### PR TITLE
Introducing API gateway

### PR DESCRIPTION
**Pull request by ggarri**

### CODE REVIEW

File: http-gateway.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+curl -X POST https://api.gateway.lightstreams.network/user/signup \
+  -H 'Content-Type: application/json' -d '{"password":"password"}'
+```
+
+Visit the following [link](/http-api-doc) to learn how about every
+available entrypoint.
+
+## Community
+
+Whether you have any question or request related to our Gateway we will
```
Comment (azappella) [inline http-gateway.md]: "If you have any questions..." instead of "Whether"

File: http-gateway.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+<br>
+
+> IMPORTANT: Be aware that we are currently working on the development
+of this APIand therefore the environment might suffer short outages and
+data could be removed without beforehand notice.
+
+# Introduction
+
+In order to facilitate the on-boarding of new developer to our
```
Comment (azappella) [inline http-gateway.md]: onboarding

Comment (azappella) [inline http-gateway.md]: new developers

File: http-gateway.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+[leth SDK](/getting-started-with-lightstreams-node) we provide an open
```
Comment (azappella) [inline http-gateway.md]: [lightstreams node] instead of [leth sdk]

File: index.md [hunk 4]
```diff
@@ -17,3 +17,4 @@ We are committed to providing a user-friendly and easy adoption path for users,
 - [Overview](overview)
 - [How does it work?](how-does-it-work)
 - [Getting started with Lightstreams Node](getting-started-with-lightstreams-node)
+- [Try-out out HTTPs Gateway](http-gateway)
```
Comment (azappella) [inline index.md:20]: "Try the HTTP API " instead

File: http-gateway.md (hunk 1 above)
Comment (ggarri) [reply http-gateway.md]: Oki

File: http-gateway.md (hunk 2 above)
Comment (ggarri) [reply http-gateway.md]: oki

**Pull request merged**